src/lib.rs

```diff
@@ -1102,16 +1102,14 @@ pub unsafe trait TryFromBytes {
         candidate: Maybe<'_, Self, A>,
     ) -> bool;
 
-    /// Attempts to interpret a byte slice as a `Self`.
-    ///
-    /// `try_ref_from` validates that `bytes` contains a valid `Self`, and that
-    /// it satisfies `Self`'s alignment requirement. If it does, then `bytes` is
-    /// reinterpreted as a `Self`.
+    /// Attempts to interpret the given `candidate` as a `&Self` without
+    /// copying.
     ///
-    /// Note that Rust's bit validity rules are still being decided. As such,
-    /// there exist types whose bit validity is ambiguous. See
-    /// [here][TryFromBytes#what-is-a-valid-instance] for a discussion of how
-    /// these cases are handled.
+    /// If the bytes of `candidate` are a valid instance of `Self`, this method
+    /// returns a reference to those bytes interpreted as `Self`. If
+    /// `candidate.len() < size_of::<Self>()` or `candidate` is not aligned to
```

Comment:
This supports unsized types, so `candidate.len() < size_of::<Self>()` is too restrictive.

Reply:
This is the phrasing we use on `FromBytes` which, iirc, we selected because it's roughly correct and the actual property (especially for suffix casts) is really hard to explain. Do you have a suggestion for an alternative phrasing we should be using on these methods?

Reply:
I've updated #29 to block on this instead of blocking this PR.